.github/templates/docker_context/action.yml

```
@@ -15,5 +21,5 @@ runs:
   using: "composite"
   steps:
     - id: environment-generator
-      run: ${{ github.action_path }}/docker_context.sh
+      run: ${{ github.action_path }}/docker_context.sh "${{inputs.modified_modules}}"
```

could you input this as an env variable instead.
```suggestion
      run: ${{ github.action_path }}/docker_context.sh
      env:
          MODIFIED_MODULES: "${{inputs.modified_modules}}"
```

---

Done